app/src/components/SingleDatePicker/SingleDatePicker.js

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Popover, GU, RADIUS, useTheme } from '@aragon/ui'
+import DatePicker from './DatePicker'
+import Labels from './Labels'
+import { SINGLE_DATE } from './consts'
+import { dayjs, dateFormat } from '../../utils/date-utils'
+import handleSingleDateSelect from './utils'
+
+function SingleDatePicker({ format, startDate: startDateProp }) {
```

Comment:
I think we need to have `onChange` here that is the handler function that `AddEmployee` will pass to this component so we can call it once the date is changed. And we should probably call it in `handleDateClick `
```suggestion
function SingleDatePicker({ format, onChange, startDate: startDateProp }) {
```